src/components/global/Header.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useTheme } from "../../context";
+
+export default function Header() {
+  const { theme, setTheme } = useTheme();
+
+  return (
+    <div className="App">
+      <nav
+        style={{ position: "relative" }}
+        className={`top-nav flex align-items-center justify-between ${
+          theme === "light" ? "bg-slate-900" : "bg-orange-200"
+        }`}
+      >
+        <input id="top-nav-toggle" type="checkbox" />
+        <Link to="/" className="text-red-500 flex align-items-center gap-2">
+          <i className="fa-solid fa-fire text-4xl "></i>
+          <h2 className="text-3xl">
+            <span className="text-orange-500 text-4xl">CAMP</span>fire
+          </h2>
+        </Link>
+
+        <ul
+          className="links gap-10 text-2xl flex align-items-center justify-center"
+          style={{ opacity: 1 }}
+        >
+          <li>
+            <button
+              onClick={() => {
+                if (theme === "light") {
+                  setTheme("dark");
+                }
+                if (theme === "dark") {
```

Comment:
There are two if statements here. Instead of that, you can use else if for the second one. Putting multiple ifs will cause javascript to check for all the conditions, but else if will make it check only the condition which is true.